.gitignore

```diff
@@ -21,6 +21,14 @@ dev/logs/
 dev/connections.yaml
 dev/dags/
 
+# Integtation Tests
```

Comment:
correct the typo here

Reply:
Fixed 48f2b56790cc598041ed92114f377176d0e1f556